source/include/boot_upgrade_package.h

```diff
@@ -6,13 +6,17 @@
 
 typedef struct
 {
-    uint32_t dummy;
-    /* TODO all fields here */
+ 
+    uint8_t prefix[4];
+    uint32_t version;
+    uint32_t size;
+    uint8_t signature[64]
```

Comment:
We are planning to use RSA-1024 for now please increase the array size to 128 (=1024/8)